shp2osm/taglist.csv

```diff
@@ -1,99 +1,97 @@
-Name,Key,Value,Key2,value2,key3,value3
-Shoemaker,Craft,shoemaker,,,,
-water well with no pump(man-made),Man_made,water_well,,,,
-Copy shop,Shop,Copyshop,noname,yes,,
-Alcohol shop,Shop,Alcohol,,,,
-motorcycle repair / mechanic,shop,motorcycle,motorcycle:repair,yes,,
-Motorcycle parts,Shop, motorcycle,,,,
-Motorcycle tyres,Shop,tyres,,,,
-Agrarian shop,Shop,agrarian,agrarian,*,name,*
-pharmacy/dispensary,amenity,pharmacy,,,,
-Doctor,amenity,doctors,,,,
-herbalist,shop,herbalist,,,,
-Borehole,Man_made,water_well,,,,
-water well with pump,Man_made,water_well,,,,
-Football field,Public,Football,,,,
-Mobile phone seller,Shop,mobile_phone,,,,
-bakery,Craft,bakery,,,,
-grinding mill (building),Craft,grinding_mill,,,,
-grinding mill (landuse),Craft,grinding_mill,,,,
-oil mill,Craft,oil_mill,,,,
-Car Wash,amenity,car_wash,,,,
-electrician / electronic repair,Craft,electronics_repair,,,,
-welder / metal worker,Craft,welder,,,,
-tailor / dress maker,Craft,Clothes,,,,
-car repair / mechanic,shop,car_repair,,,,
-vulcanizer / tyre repair,service,tyres,,,,
-carpenter / wood worker,Craft,Carpenter,,,,
-Boat builder,Craft,boatbuilder,,,,
-land used for a school,amenity,School,,,,
-Any building,General,Building,,,,
-A building with healthcare of unknown purpose (not a shop selling medicine) ,Health,healthcare,,,,
-Dispensary selling medicine,Health,drugs,,,,
-telecommunication mast (man-made),man_made,mast,,,,
-Restaurant,amenity,restaurant,,,,
-bar(drinks only),amenity,bar,,,,
-Barber shop,Shop,hairdresser,male,yes,,
-beauty salon,Shop,hairdresser,female,yes,,
-Internet cafe,amenity,internet_cafe,,,,
-Photograph shop,Shop,Electronic,internet_access,*,,
-Motorcycle services,Shop,Car/Motorcycle,,,,
-Frozen yoghurt shop,amenity,ice_cream,name,*,,
-Convenience shop/mini-mart,Shop,convenience,,,,
-fuel shop (They are often just a house with some barrels or bottles clearly displayed.),Shop,Fuel,,,,
-Fuel station,amenity,Fuel,,,,
-A building that is a shed,General,Building,,,,
-A building that has no walls,General,Building,,,,
-Residential road,General,Road,,,,
-Path,General,Road,,,,
```

Comment:
@jeafreezy 
I deleted road options, because these cannot be a "node" feature. Do you agree?

Reply:
Yes, I agree.